pocs/scheduler/constraint.py

```diff
@@ -170,10 +170,10 @@ def get_score(self, time, observer, observation, **kwargs):
         score = self._score
 
         target = observation.field
-        observed = observation.observed_list
+        observed_list = kwargs['observed_list']
```

Comment:
For normal operations you'll still want to pull from `observation.observed_list` as that's how the scheduler builds it up. So do a`kwargs.get('observed_list', observation.observed_list)` here.